sda-commons-dependencies/build.gradle

```diff
@@ -99,10 +105,8 @@ dependencies {
     api "org.jboss.logging:jboss-logging:3.4.1.Final"
     api "org.jboss.weld.se:weld-se-core:$weldVersion"
     api "org.jboss.weld.servlet:weld-servlet-core:$weldVersion"
-    api "org.objenesis:objenesis:3.1" // not 3.x because we have a version conflict with Mockito
```

Comment:
Why is this comment gone, is the conflict resolved? If this is so, we can remove this lone completly.

Reply:
I removed the managed dependency. We now get version 2.6 defined by dropwizard-bom. Seems to be the same version for Mockito.

Reply:
\o/